pkg/version/base.go

```diff
@@ -51,7 +51,7 @@ var (
 	// semantic version is a git hash, but the version itself is no
 	// longer the direct output of "git describe", but a slight
 	// translation to be semver compliant.
-	gitVersion   string = "v0.0.0-master+$Format:%h$"
+	gitVersion   string = "v1.5.3-beta.0+$Format:%h$"
```

Comment:
hm, it's weird that client go needs this import. I have to look more carefully, not sure if it makes sense to set this or not.